.changeset/tall-chicken-repeat.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris': major
+---
+
+Removed Summer Editions experimental styles and code from Avatar component
```

Comment:
should we make changelogs for all of these that we do?

Reply:
Good point 🤔  I did this to help test the **Version Packages (beta)** PR which creates the prerelease of `next`. However, I think what we could do is just update a single changeset with a list of components included for that release.

So for the next PR removing uplift code, it would just amend and adjust this changeset 🤷 seems easier than listing out a bunch of changesets for essentially the same change but different components